x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -67,6 +65,49 @@ role descriptor with no privileges. The derived API key can be used for
 authentication; it will not have authority to call {es} APIs.
 
 --
++
+`applications`::: (list) A list of application privilege entries.
+`application` (required):::: (string) The name of the application to which this entry applies
+`privileges` (required):::: (list) A list of strings, where each element is the name of an application
+privilege or action.
+`resources` (required):::: (list) A list resources to which the privileges are applied.
+
+`cluster`::: (list) A list of cluster privileges. These privileges define the
+cluster level actions that API keys are able to execute.
+
+`global`::: (object) An object defining global privileges. A global privilege is
+a form of cluster privilege that is request-aware. Support for global privileges
+is currently limited to the management of application privileges.
+This field is optional.
+
+`indices`::: (list) A list of indices permissions entries.
```

Comment:
Good catch. I've tried to be consistent with the [existing example](https://github.com/elastic/elasticsearch/blob/0ce155bb1669a9f3c2a9fc0060375e8f8a7cdc2b/x-pack/docs/en/rest-api/security/create-api-keys.asciidoc?plain=1#L136-L141) and did not notice it uses `index` instead `indices`. Technically,  both `indices` and `index` are accepted  [when role descriptor is parsed](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/RoleDescriptor.java#L491C10-L492)), but I think `indices` should be preferable since it's the only documented and used in [create/update role examples](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-put-role.html).